<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CreateMenu</class>
 <widget class="QDialog" name="CreateMenu">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>400</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>600</width>
    <height>350</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Create menu</string>
  </property>
  <property name="modal">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>661</width>
     <height>351</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="m_mainLayout">
    <item>
     <layout class="QHBoxLayout" name="m_ListLayout">
      <item>
       <layout class="QVBoxLayout" name="m_recipeListLayout">
        <item>
         <widget class="QLabel" name="m_recipesLabel">
          <property name="text">
           <string>Recipes:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QListView" name="m_recipeListView"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="m_selectedRecipesLayout">
        <item>
         <widget class="QLabel" name="m_selectedRecipesLabel">
          <property name="text">
           <string>Selected recipes:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QListView" name="m_selectedRecipesListView"/>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="m_tableLayout">
      <item>
       <widget class="QLabel" name="m_ingredientsLabel">
        <property name="text">
         <string>Ingredients:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTableView" name="m_ingredientsTableView"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="m_buttonLayout">
      <property name="spacing">
       <number>10</number>
      </property>
      <item>
       <widget class="QPushButton" name="m_addButton">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>&amp;Add</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="m_removeButton">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>&amp;Remove</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="m_cancelButton">
        <property name="text">
         <string>&amp;Cancel</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
